frontend/src/components/pages/home-page.tsx

```diff
@@ -441,9 +443,8 @@ const MarimoFileComponent = ({
       </div>
       <div className="flex flex-col gap-1 items-end">
         <div className="flex gap-3 items-center">
-          <div>
-            <SessionShutdownButton filePath={file.path} />
-          </div>
+          <CopyNotebookButton filePath={file.path} />
+          <SessionShutdownButton filePath={file.path} />
```

Comment:
I don't think we need this on the homepage - at least for now. 

This is also probably why you were getting mismatch sessions. Since the homepage doesn't have an active session